packages/expandable/tests/Expandable.test.tsx

```diff
@@ -1,66 +1,69 @@
+import userEvent from "@testing-library/user-event";
 import { createSerializer } from "@emotion/jest";
-import toJson from "enzyme-to-json";
 
 import { Expandable } from "../";
 
 expect.addSnapshotSerializer(createSerializer());
 
 describe("Sidebar", () => {
   it("renders", () => {
-    const component = mount(
+    const { asFragment } = render(
       <Expandable label="Label" isOpen={true}>
         Content
       </Expandable>
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
-  it("toggles open and closed", () => {
-    const component = mount(
-      <Expandable label="Label" isOpen={true}>
-        Content
-      </Expandable>
+  it("toggles open and closed", async () => {
+    const { getByText, queryByText, getByTestId } = render(
+      <Expandable label="Label">Content</Expandable>
     );
 
-    expect(component.find("button").props()["aria-expanded"]).toBe(true);
-    component.find("button").simulate("click");
-    expect(component.find("button").props()["aria-expanded"]).toBe(false);
+    expect(queryByText("Content")).not.toBeVisible();
+    const labelEl = getByText("Label");
+    labelEl.click();
+    getByText("Content");
+    labelEl.click();
+    expect(queryByText("Content")).not.toBeVisible();
+
+    const chevronIcon = getByTestId("icon");
+    await userEvent.click(chevronIcon);
+    getByText("Content");
+    await userEvent.click(chevronIcon);
+    expect(queryByText("Content")).not.toBeVisible();
   });
 
   it("fires onChange when button is clicked", () => {
     const onChange = jest.fn();
-    const component = mount(
+    const { getByText } = render(
       <Expandable label="Label" isOpen={true} onChange={onChange}>
         Content
       </Expandable>
     );
-
     expect(onChange).not.toHaveBeenCalled();
 
-    component.find("button").simulate("click");
+    const labelEl = getByText("Label");
+    labelEl.click();
 
-    expect(onChange).toHaveBeenCalledTimes(1);
-    const openArgument = onChange.mock.calls[0][0];
-    expect(openArgument).toBe(false);
+    expect(onChange).toHaveBeenCalledWith(false);
+
+    labelEl.click();
+
+    expect(onChange).toHaveBeenLastCalledWith(true);
   });
 
   it("toggles open and closed based on controlledIsOpen prop when passed", () => {
-    const component = mount(
+    const { getByText } = render(
       <Expandable label="Label" controlledIsOpen>
         Content
       </Expandable>
     );
 
-    expect(component.find("button").props()["aria-expanded"]).toBe(true);
-    component.find("button").simulate("click");
-    expect(component.find("button").props()["aria-expanded"]).toBe(true);
-
-    component.setProps({
-      controlledIsOpen: false
-    });
-    component.update();
-
-    expect(component.find("button").props()["aria-expanded"]).toBe(false);
+    getByText("Content");
+    const labelEl = getByText("Label");
+    labelEl.click();
+    getByText("Content");
```

Comment:
I think we should assert something in each test. Why not `expect(queryByText("Content")).toBeVisible();`?

If that's not preferable, what about at least wrapping it in `expect(getByText("Content").not.toThrow()`

Failing tests without assertions are harder to debug why it's happening.

Reply:
@vacas5 getBy queries are in itself assertions and if not found it automatically shows an error showing an element with the following attribute is not found in the jsDom and prints the jsDom.

Reply:
The issue is just the readability of the tests themselves. Having assertions is a signal to readers that a desired state is achieved and it is something we are testing for and want to prove. After digging into the code I'd suggest something like `expect(getByText("Content")).toBeInTheDocument();` rather than my other suggestions. Even though it may seem superfluous you're telling any future readers that an important assertion was made (otherwise they just might skim right over it)

Reply:
I agree with the inclusion of assertions for readability. This all looks good to me otherwise. If you fix that and pull in main it will be a ✅ from me.